src/main/java/mate/controller/manufacturer/CreateManufacturerController.java

```diff
@@ -0,0 +1,36 @@
+package mate.controller.manufacturer;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.lib.Injector;
+import mate.model.Manufacturer;
+import mate.service.ManufacturerService;
+
+@WebServlet(urlPatterns = "/manufacturers/create")
+public class CreateManufacturerController extends HttpServlet {
+    private static final Injector injector =
+            Injector.getInstance("mate");
+    private final ManufacturerService manufacturerService =
+            (ManufacturerService) injector.getInstance(ManufacturerService.class);
+
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        req.getRequestDispatcher("/WEB-INF/views/manufacturers/create.jsp").forward(req, resp);
+    }
+
+    @Override
+    protected void doPost(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        Manufacturer manufacturer = new Manufacturer();
+        String name = req.getParameter("manufacturer_name");
+        String country = req.getParameter("manufacturer_country");
+        manufacturer.setName(name);
+        manufacturer.setCountry(country);
+        manufacturerService.create(manufacturer);
+    }
```

Comment:
You should add a redirect after creating a Manufacturer. Otherwise, the user will stay on the same page without any feedback.